lib/statistics.js

```diff
@@ -1,43 +1,33 @@
 'use strict';
 
-const Stats = require('fast-stats').Stats;
-
-function percentileName(percentile) {
-  if (percentile === 0) {
-    return 'min';
-  } else if (percentile === 100) {
-    return 'max';
-  } else if (percentile === 50) {
-    return 'median';
-  }
-  return `p${String(percentile).replace('.', '_')}`;
-}
-
 class Statistics {
   constructor() {
-    this.stats = new Stats();
+    this.values = [];
   }
 
   add(value) {
-    this.stats.push(value);
+    this.values.push(value);
     return this;
   }
 
-  summarize(options) {
-    if (isNaN(this.stats.amean())) {
+  summarize() {
+    const values = this.values;
```

Comment:
Is this assignment meant to avoid modifying this.values (which I think is a good idea!)? If so it needs to be ```const values = Array.from(this.values);``` since objects (including arrays) in javascript have reference semantics, so it's referencing the same array.